build.gradle

```diff
@@ -285,7 +285,11 @@ task integrationTest(type: Test) {
     systemProperty "java.util.logging.manager", "org.apache.logging.log4j.jul.LogManager"
     testClassesDirs = sourceSets.integrationTest.output.classesDirs
     classpath = sourceSets.integrationTest.runtimeClasspath
-
+    retry {
+        failOnPassedAfterRetry = false
+        maxRetries = 2
```

Comment:
Actually, these retries were needed because tests related to snapshot creation were unstable. I manage to correct this and the correction is included in the current PR, please see: https://github.com/opensearch-project/security/pull/2153/commits/cdc87802914b6d7f1ae0a2feeb88c699fa45e2d6 
Therefore I hope that two retries are more than enough :)